content/en/real_user_monitoring/browser/advanced_configuration.md

```diff
@@ -18,10 +18,87 @@ further_reading:
       text: 'Tracking custom user actions'
 ---
 
-## Control sensitive RUM data
-If your RUM data contains sensitive information that need redacting, configure the Browser SDK to redact sensitive sequences, or to discard selected RUM events, by using the `beforeSend` callback when you initialize RUM.
+## Enrich and control RUM data
+The RUM SDK automatically captures most RUM events and populates their main attributes. The `beforeSend` callback function gives you access to every event collected by the RUM SDK before they get sent to Datadog. Intercepting the RUM events allows you to:
+* Enrich your RUM events with extra context attributes
+* Modify your RUM events to modify their content, or redact sensitive sequences
```

Comment:
```suggestion
* Redact sensitive sequences of your RUM events
```
We still can't modify the content of the event, only some allowed fields

Reply:
Let me add a note "(see list of editable attributes[link])"